packages/mui-material/src/styles/createPalette.d.ts

```diff
@@ -129,6 +129,58 @@ export interface PaletteWithChannels {
   background: TypeBackground;
   getContrastText: (background: string) => string;
   augmentColor: (options: PaletteAugmentColorOptions) => PaletteColor;
+  // component tokens
+  AppBar: {
+    defaultBgColor: string;
+  };
+  Chip: {
+    defaultBorderColor: string;
+    defaultAvatarColor: string;
+    defaultIconColor: string;
+  };
+  FilledInput: {
+    bgColor: string;
+    hoverBgColor: string;
+    disabledBgColor: string;
+  };
+  LinearProgress: {
+    primaryBgColor: string;
+    secondaryBgColor: string;
+    errorBgColor: string;
+    infoBgColor: string;
+    successBgColor: string;
+    warningBgColor: string;
+  };
+  Slider: {
+    primaryTrack: string;
+    secondaryTrack: string;
+    errorTrack: string;
+    infoTrack: string;
+    successTrack: string;
+    warningTrack: string;
+  };
+  SnackbarContent: {
+    bgColor: string;
+  };
+  StepConnector: {
+    borderColor: string;
+  };
+  StepContent: {
+    borderColor: string;
+  };
+  Switch: {
+    defaultColor: string;
+    defaultDisabledColor: string;
+    primaryDisabledColor: string;
+    secondaryDisabledColor: string;
+    errorDisabledColor: string;
+    infoDisabledColor: string;
+    successDisabledColor: string;
+    warningDisabledColor: string;
+  };
+  TableCell: {
+    borderColor: string;
```

Comment:
```suggestion
    border: string;
```
It is a key in the palette, we know it's color

Reply:
Done.